docs/getting_started/example_configurations.md

```diff
@@ -28,11 +34,22 @@ Following benchmark [configuration]({{page.path_to_root}}benchmark_configuration
 
 **[benchmark-hotrod-rest-memcached.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-hotrod-rest-memcached.xml)** - Shows performance testing for same-server/different-client configurations
 
+> Environment variables:
+> * *ISPN_8_HOME*	- pointing to Infinispan home directory
+
 **[benchmark-hotrod-spark-mapreduce.xml](https://github.com/radargun/radargun/blob/master/extensions/mapreduce/src/main/resources/benchmark-hotrod-spark-mapreduce.xml)** - Shows how to benchmark MapReduce operations on Infinispan coupled with Spark
 
+> Environment variables:
+> * *SPARK_HOME*	- pointing to Spark home directory
+> * *ISPN_82_HOME*	- pointing to Infinispan 8.2 home directory
+
 **[benchmark-hotrod-streaming.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-hotrod-streaming.xml)** - Shows how to compare memory usage on Infinispan clients using and not using Streaming API
 
-**[benchmark-iteration.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-iteration.xml)** - Shows how to use `iterate` stage to iterate over all entries in the cache
+> Environment variables:
+> * *ISPN_9_ZIP_PATH*	- path to file with Infinispan 9 server
```

Comment:
I would suggest examples should show as many configuration variations as possible. Having zip is the best approach to making sure you have a clean installation every time, it would be pity if we did not have example on how to configure it.